docs/static/readme.md

```diff
@@ -0,0 +1 @@
+This is the ud-viz documentation
```

Comment:
Why is docs/static/Installation.md an empty file ? If it's a dummy for e.g. jsdoc can't this be stated in that file ?